docs/build-apps/build-properties.md

```diff
@@ -0,0 +1,32 @@
+---
+title: .NET for iOS, Mac Catalyst, macOS, and tvOS Build Properties
+description: .NET for iOS, Mac Catalyst, macOS, and tvOS Build Properties
+ms.date: 09/19/2024
+---
+
+# Build Properties
+
+MSBuild properties control the behavior of the
+[targets](build-targets.md).
+They're specified within the project file, for example **MyApp.csproj**, within
+an MSBuild PropertyGroup.
+
+## MaciOSPrepareForBuildDependsOn
+
+A semi-colon delimited property that can be used to extend the build process.
+MSBuild targets added to this property will execute early in the build for both
+application and library project types. This property is empty by default.
+
+Example:
+
+```xml
+<PropertyGroup>
+  <MaciOSPrepareForBuildDependsOn>BeforeBuild</MaciOSPrepareForBuildDependsOn>
+</PropertyGroup>
+
+<Target Name="BeforeBuild" >
```

Comment:
"BeforeBuild" might be an unfortunate name for an example target, because it's already exists to do things before the build.

Maybe something like "BeforeMaciOSBuild"?

Reply:
Updated this to `MyCustomTarget`..